.github/workflows/check-readmes.yaml

```diff
@@ -10,6 +10,13 @@ jobs:
       - name: Check out code
         uses: actions/checkout@v4
 
+      - name: Install oc
+        run: |
+          url=https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest-4.17/openshift-client-linux.tar.gz
+          if ! which oc; then
+            sudo bash -c "curl -sL "$url" | tar -xz -C /usr/bin oc"
```

Comment:
You do not need to run `curl` (neither the `xz` decompressor) through `sudo`.  Only the write to `/usr/bin/` requires root privileges.  You should also run `curl` with `--fail` so that it provides a meaningful error message when the URL starts to return `404` etc.

Reply:
PTAL

Reply:
I think it can be further simplified/secured like this:
```sh
curl --fail -sL "$url" | gzip -cd | sudo -- tar -x -C /usr/bin oc
```

Reply:
> I think it can be further simplified/secured like this:
> 
> ```shell
> curl --fail -sL "$url" | gzip -cd | sudo -- tar -x -C /usr/bin oc
> ```

Cool. But, why can't keep `z` option for tar?

Reply:
You can but in that case `gzip` will unnecessarily run with `root` privileges.

Reply:
Updated